examples/shader-coverage-bvh-traversal/vk_compute_demo.cpp

```
@@ -0,0 +1,416 @@
+    VkWriteDescriptorSet w = {};
+    w.sType = VK_STRUCTURE_TYPE_WRITE_DESCRIPTOR_SET;
+    w.dstSet = set;
+    w.dstBinding = binding;
+    w.descriptorCount = 1;
+    w.descriptorType = VK_DESCRIPTOR_TYPE_STORAGE_BUFFER;
+    w.pBufferInfo = &dbi;
+
+    vkUpdateDescriptorSets(device, 1, &w, 0, nullptr);
+}
+
+void Context::dispatch(
```

Cppcheck (2.20.0)

[style] The function 'dispatch' is never used.

(unusedFunction)